packages/material-ui-codemod/src/v5.0.0/jss-to-styled.js

```diff
@@ -7,7 +7,7 @@ export default function transformer(file, api, options) {
   const root = j(file.source);
 
   function getFileNameWithoutExt() {
-    return (file.path.split('/').slice(-1)[0] || '').replace(/^([^.]*)\.(.*)/, '$1');
+    return (file.path.split(/(\\|\/)/g).slice(-1)[0] || '').replace(/^([^.]*)\.(.*)/, '$1');
```

Comment:
This could be simplified to:

```suggestion
    return path.basename(file.path).replace(/^([^.]*)\.(.*)/, '$1');
```

or even
```suggestion
    return path.basename(file.path, path.extname(file.path));
```

But the behavior would be different for filenames containing multiple dots.

Reply:
We can't include names with multiple dots as it won't be a valid class name. Let me add the first suggestion then.